ENGAGEHF/Education/Views/VideoListSection.swift

```diff
@@ -0,0 +1,104 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct VideoListSection: View {
```

Comment:
I would suggest that we use a white card layout with a system grouped background here as we do for the rest of the app. I would suggest that we document this and address this in a general styling PR once we incorporated all the different major features in the app.